services/settings/pkg/store/defaults/defaults.go

```diff
@@ -94,6 +122,17 @@ func generateBundleAdminRole() *settingsmsg.Bundle {
 			DeleteProjectSpacesPermission(All),
 			DeleteReadOnlyPublicLinkPasswordPermission(All),
 			DisableEmailNotificationsPermission(Own),
+			ProfileEmailSendingIntervalPermission(Own),
+			ProfileEventShareCreatedPermission(Own),
+			ProfileEventShareRemovedPermission(Own),
+			ProfileEventShareExpiredPermission(Own),
+			ProfileEventSpaceSharedPermission(Own),
+			ProfileEventSpaceUnsharedPermission(Own),
+			ProfileEventSpaceMembershipExpiredPermission(Own),
+			ProfileEventSpaceDisabledPermission(Own),
+			ProfileEventSpaceDeletedPermission(Own),
+			ProfileEventPostprocessingStepFinishedPermission(Own),
+			ProfileEventScienceMeshInviteTokenGeneratedPermission(Own),
```

Comment:
Question: Do we need a permission for every notification setting? Can't we just have one permission that allows changing all of them?

Reply:
It is not possible now